index.js

```diff
@@ -320,7 +320,9 @@ export const setup = (build, _options) => {
     /** @type {[string, string][]} */
     const filesToBuild = [];
     warnMetafile();
-    const cssOutputsMap = Object.entries(r.metafile?.outputs ?? {}).reduce((m, [o, { inputs }]) => {
+
+    const cssOutputsMap = Object.entries(r.metafile?.outputs ?? {})
+        .reduce(/** @type {(m: Record<string, string>, o: any) => Record<string, string>} */(m, [o, {inputs}]) => {
```

Comment:
Removing a typescript failure due to it not understanding the type of the accumulator. There could be a better solution to be quite honest.